WebClient/src/main/java/service/home/HelloWorld.java

```diff
@@ -1,29 +0,0 @@
-package service.home;
-
-import model.BookData;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
-import parser.ebookscom.EBookCategory;
-import robot.ebooks.EBooksRobotManager;
-
-import java.util.List;
-import java.util.Map;
-
-/**
- * Created by ehsan on 26.08.16.
```

Comment:
I have a feeling I have been conditioned. If I see a name without Uppercase, I'm thinking of Ehsan.